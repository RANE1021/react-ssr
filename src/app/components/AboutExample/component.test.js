import React from "react";
import { shallow } from "enzyme";
import { renderer } from "react-test-renderer"

import Example from "./component";

const styles = {};

describe("The about component mounts with styles and three divs", () => {
  const shallowExample = shallow(<Example styles={styles} />);
  it("loads with divs", () => {
    expect(shallowExample.find("div").length).toBe(3);
  });

  it('matches previous snapshot', () => {
    expect(Example).toMatchSnapshot()
  })
});
