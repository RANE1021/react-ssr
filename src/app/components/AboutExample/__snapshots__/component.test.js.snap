// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The about component mounts with styles and three divs matches previous snapshot 1`] = `[Function]`;
